06-building-a-program-start-to-finish/writing-tests/output/InvertedIndex.spec.js

```diff
@@ -0,0 +1,292 @@
+const invertedIndex = new InvertedIndex();
+
+describe('InvertedIndex class',
+ () => {
+   it('should create an instance of itself on instantiation',
+  () => expect(invertedIndex instanceof InvertedIndex).toBeTruthy());
+ });
+
+describe('updateIndexedFilesRecords method',
+ () => {
+   let books;
+   let file;
+   let file2;
+   beforeEach(() => {
+     books = { 'touch.json': {
```

Comment:
You have also not implemented this feedback from Bayo.... **_(Please read this data from an actual JSON file outside this class.)_** Why?